src/test/java/com/google/devtools/build/lib/syntax/EvaluationTest.java

```diff
@@ -213,6 +213,31 @@ public void testFloorDivision() throws Exception {
         .testIfExactError("integer division by zero", "5 // 0");
   }
 
+  @Test
+  public void testBitwiseOperations() throws Exception {
```

Comment:
Let's add all new tests in the testdata/*.star files where possible; the tests written in Java have no advantage yet tend to be more verbose, with more fiddly quoting, and they require a recompile each time one is changed.